troubleshooting/known_issues/orphaned_cfs_pods.md

```diff
@@ -0,0 +1,34 @@
+# Orphaned CFS Pods After Booting or Rebooting
+
+After a boot or reboot a few CFS Pods may continue running even after they've
+finished and never go away.
+The state of these Pod is that the only container still running in the Pod is
```

Comment:
Change to:

These CFS pods are in a state where the only container running in the pod is `istio-proxy`, and the pod does not have a `metadata.ownerReference` value.